src/components/projects/CreateProjectForm.vue

```diff
@@ -518,9 +517,8 @@ export default {
             ],
 
             project_constraints: {
-                project_is_active: false,
+                project_is_active: true,
```

Comment:
Is this change specific to your feature, or is this a change specifically so that projects are by default put into the pool of projects that run automatically?

Reply:
There was an inconsistency that a project was not active upon loading the page, but was active after clicking the Clear Form button. This didn't really make sense, so we chose to default to active.